.storybook/themes/news.scss

```diff
@@ -812,6 +812,53 @@
 					list-style-position: inside,
 					list-style-type: disc,
 				),
+				author-bio: (
+					gap: var(--global-spacing-4),
+				),
+				author-bio-author: (
+					gap: var(--global-spacing-4),
+					display: flex,
+					flex-direction: row,
+					components: (
+						image: (
+							max-width: var(--global-spacing-12),
+							max-height: var(--global-spacing-12),
+							border-radius: var(--global-border-radius-2),
+						),
+						stack: (
+							flex: 1,
```

Comment:
I feel like this should have the `--global-spacing-2` gap here, but the design calls for the `desktop` to have an additional `1rem` gap under the description.  That could potentially just be a bottom margin of `global-spacing-4` to add on to the `--global-spacing-2` to make the necessary `1.5rem`